poucave/html/script.js

```diff
@@ -25,10 +25,24 @@ async function refreshCheck(check, options = {}) {
     }
   }
 
+  // In SVG diagram too :) 
+  const svgObject = document.getElementById('svg-diagram').contentDocument;
+  const svgElement = svgObject ? svgObject.getElementById(`${check.project}--${check.name}`) : null;
```

Comment:
A string made up of "{project}--{name}" is brittle.  It is not obvious this relationship between project/check name and values baked into an SVG.  Maybe a warning in the documentation is enough to save someone (us!) some debugging time in the future.